<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSamplePlayerClass</class>
 <widget class="QMainWindow" name="QtSamplePlayerClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>libdash player</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>781</width>
         <height>591</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="cb_mpd">
          <item>
           <property name="text">
            <string>http://www-itec.aau.at/~cmueller/libdashtest/showcases/big_buck_bunny_480.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://www-itec.aau.at/~cmueller/libdashtest/showcases/redbull_2_resolutions.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://www-itec.aau.at/~cmueller/libdashtest/showcases/redbull_segment_template.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/1a/netflix/exMPD_BIP_TC1.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/1a/sony/SNE_DASH_SD_CASE1A_REVISED.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/1b/envivio/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/1b/thomson-networks/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/1c/envivio/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/2a/envivio/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/2a/sony/SNE_DASH_CASE_2A_SD_REVISED.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/2a/thomson-networks/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/3a/fraunhofer/ed.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/3b/fraunhofer/elephants_dream_heaac2_0.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/3b/sony/SNE_DASH_CASE3B_SD_REVISED.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/4b/sony/SNE_DASH_CASE4B_SD_REVISED.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/5a/1/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/5b/1/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/6c/envivio/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://dash.edgesuite.net/dash264/TestCases/6c/envivio/manifest2.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://tvnlive.dashdemo.edgesuite.net/live/manifest.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-full-gdr/mp4-full-gdr-mpd-AV-BS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-full-gdr/mp4-full-gdr-mpd-AV-NBS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-live-periods/mp4-live-periods-mpd.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-live/mp4-live-mpd-AV-BS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-live/mp4-live-mpd-AV-NBS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-main-multi/mp4-main-multi-mpd-AV-NBS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-main-ogop/mp4-main-ogop-mpd-AV-BS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-main-ogop/mp4-main-ogop-mpd-AV-NBS.mpd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>http://ftp-itec.uni-klu.ac.at/dash/conformance/mp4-onDemand/mp4-onDemand-mpd-AV.mpd</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_mpd">
          <property name="text">
           <string>http://www-itec.aau.at/~cmueller/libdashtest/showcases/big_buck_bunny_480.mpd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="sampleplayer::renderer::QTGLRenderer" name="videoelement" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <widget class="QSlider" name="progressSlider_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>540</y>
             <width>799</width>
             <height>19</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="progressSlider">
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="button_stop">
       <property name="geometry">
        <rect>
         <x>173</x>
         <y>623</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_start">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>623</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_pause">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>623</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>pause</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>620</y>
         <width>141</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="bitmovinLogo">
          <property name="maximumSize">
           <size>
            <width>182</width>
            <height>49</height>
           </size>
          </property>
          <property name="text">
           <string>Video Segment Index :</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="libdashLogo">
          <property name="maximumSize">
           <size>
            <width>182</width>
            <height>62</height>
           </size>
          </property>
          <property name="text">
           <string>Audio Segment Index :</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>620</y>
         <width>98</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Current profile:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_profile">
          <property name="text">
           <string>default</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>261</x>
         <y>621</y>
         <width>171</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Downloading segment speed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_segment_speed">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="w_controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="w_boxes" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>800</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_mpd">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Download MPD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_period">
            <property name="text">
             <string>Period</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_period"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Video AdaptationSet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_video_adaptationset"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Video Representation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_video_representation">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_audio_adaptationset">
            <property name="text">
             <string>Audio AdaptationSet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_audio_adaptationset"/>
          </item>
          <item>
           <widget class="QLabel" name="label_audio_representation">
            <property name="text">
             <string>Audio Representation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_audio_representation"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Video Segment Buffer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_V">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Video Frame Buffer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_VF">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Audio Segment Buffer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_A">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Audio Chunk Buffer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_AC">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="accessibleDescription">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>sampleplayer::renderer::QTGLRenderer</class>
   <extends>QWidget</extends>
   <header>Renderer/QTGLRenderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtsampleplayer.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_progressSlider_sliderReleased()</slot>
  <slot>on_progressSlider_sliderPressed()</slot>
  <slot>on_button_stop_clicked()</slot>
 </slots>
</ui>
